src/components/MapComponent.vue

```diff
@@ -0,0 +1,257 @@
+<template>
+  <div class="home">
+    <div class="options">
+      <button @click="setCreateModeTrue">createPath</button>
+      <input v-if="onCreateMode" type="text" v-model="pathName" />
+      <button v-if="onCreateMode" @click="createPolyline">Create</button>
+      <button v-if="onCreateMode" @click="cancel">Cancel</button>
+
+      <div class="listOfPaths">
+        <ol>
+          <li
+            v-for="path in paths"
+            :key="path.name"
+            class="path"
+            @click="
+              () => {
+                this.locations = path.path.path;
```

Comment:
@JatinThakur14 path.path.path is absolutely terrible object naming

Reply:
I will fix this in next patch. I was just trying to if its working or not